A-concurrency-pipeline.md

```diff
@@ -13,9 +13,9 @@ Analoginya seperti ini: bayangkan sebuah flow proses untuk auto backup database
 1. Kita perlu data *list* dari semua database yang harus di-backup, beserta alamat akses dan kredensial-nya.
 2. Kita jalankan proses backup, bisa secara sekuensial (setelah `db1` selesai, lanjut `db2`, lanjut `db3`, dst), atau secara paralel (proses backup `db1`, `db2`, `db3`, dan lainnya dijalankan secara bersamaan).
 3. Di masing-masing proses backup database sendiri ada beberapa proses yang dijalankan:
-   A. Lakukan operasi *dump* terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
-   B. File-file hasil dump kemudian di-*archive* ke bentuk `.zip` atau `.tar.gz` (misalnya).
-   C. File archive di-kirim ke server backup, sebagai contoh AWS S3.
+    * A. Lakukan operasi *dump* terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
```

Comment:
for this section, can you please adjust it like below? simply add a new line between the `3.` and `A.`.

3. Di masing-masing proses backup database sendiri ada beberapa proses yang dijalankan:

    A. Lakukan operasi dump terhadap database, outputnya berupa banyak file disimpan ke sebuah folder.
    B. File-file hasil dump kemudian di-archive ke bentuk .zip atau .tar.gz (misalnya).
    C. File archive di-kirim ke server backup, sebagai contoh AWS S3.